### DISCUSSION TITLE
Domain project depends on `MediatR` package?

### DISCUSSION THREAD

**Discussion by ngaisteve1** [jasontaylordev/CleanArchitecture#672] [Q&A]
I thought that the "Core has no dependencies on any outside layers."

But I notice that the Domain project depends on `MediatR` package.

Any thoughts?

**Comment by neman** [comment]
I do not like it either. It is clearly a copy from Ardalis template, introduced with this commit https://github.com/jasontaylordev/CleanArchitecture/commit/03862a3617bd072aa56980591650962c5aa15c26#diff-6b386feb9fd7e1e3bb38c2839ad39436c807e63707bd4a6e24cef63f6fb6f772

I will continue to use previous code, without dependencies on MediatR

**Comment by asgharb** [comment]
https://stackoverflow.com/questions/47292941/ddd-referencing-mediatr-interface-from-the-domain-project
